apps/src/p5lab/drawUtils.js

```diff
@@ -56,37 +56,43 @@ export function speechBubble(
   width,
   height,
   {
-    triangleSize = 10,
-    triangleTipX = x,
-    rectangleCornerRadius = 8,
+    tailSize = 10,
+    tailTipX = x,
+    padding = 8,
     fill = 'white',
     strokeWeight = 2,
     stroke = 'gray'
-  } = {}
+  } = {},
+  style
 ) {
   const minX = x - width / 2;
-  const minY = y - height - triangleSize;
-  const maxY = y - triangleSize;
+  const minY = y - height - tailSize;
+  const maxY = y - tailSize;
+  const tailTopX = (tailTipX + (minX + width / 2)) / 2;
```

Comment:
A comment here would help future engineers follow along with the math.

Reply:
I think there is a typo with "We take move . . .". This comment is good in that it explains what we are doing, but it's still hard to connect that to the (`tailTipX + (minX . . . .`) math. Is there a way to expand that math with named variable or add an additional comment to make to clear how we developed that equation?